TF Refactor, Raffle Variables, and Documentation

cai-michael opened the pull request:
- Raffle Variables are now set using a constructor
- TF is refactored to use JSON and dict
- Added Documentation to Methods

closes #4

tsukikita commented on Modules/tf.py:
```diff
@@ -3,11 +3,44 @@
+
+def load_tf_list():
+    """
+    Loads the TF List into the variable tfList from "Data/tfList.json"
```
Consider a bit more detail in the docstrings; i.e. I know Python is dynamically typed, but some idea of what a "tf list" is might be useful for developers. Looking at the code, I can see that it's a dictionary of species by name.

tsukikita commented on Modules/tf.py:
```diff
@@ -20,8 +53,23 @@ def writeSpeciesToCSV(name, species):
-# Gets name/species pair from file. Return 'a human' if user not in list
+def getSpecies(name):
+    """
+    Returns the name's current species
```
Similar to above, a bit more detail on the parameter would be helpful. In this case it's fairly clear that "name" should be a text value, but this isn't always the case.

tsukikita commented on Modules/rimworld.py:
```diff
@@ -40,6 +47,10 @@ def item_search(ctx):
 
 def event_search(ctx):
+    """
+    Searches the users Rimworld installation for events containing that keyword
```
"Keyword" is not an argument -- Might be worth specifying where the keyword comes from in the docstring

tsukikita commented on Modules/rimworld.py:
```diff
@@ -57,6 +68,9 @@ def event_search(ctx):
 
 def event_detail_search(ctx):
+    """
+    Returns details about a specific event
```
See above; event is not a parameter, where does this come from?

tsukikita commented on Modules/rimworld.py:
```diff
@@ -68,6 +82,9 @@ def event_detail_search(ctx):
 
 def item_detail_search(ctx):
+    """
+    Returns details about a specific item
```
Same as above

Zielmann commented:
FYI - The change to using a constructor for Raffle breaks the module import.

Exception has occurred: TypeError
__init__() takes 1 positional argument but 2 were given
  File "C:\Users\mattw\PythonProjects\TwitchBot-Public\bot.py", line 60, in <module>
    bot.load_module('Modules.raffle')

cai-michael commented:
> FYI - The change to using a constructor for Raffle breaks the module import.
> 
> Exception has occurred: TypeError
> **init**() takes 1 positional argument but 2 were given
> File "C:\Users\mattw\PythonProjects\TwitchBot-Public\bot.py", line 60, in
> bot.load_module('Modules.raffle')

Reverted these changes

Added some more documentation past Tsuki's review
